code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -63,6 +63,7 @@ var/global/list/datum/stack_recipe/sandstone_recipes = list ( \
 	icon_state = "sandbags"
 	singular_name = "sandbag"
 	force = 5
+	layer = 2.9005
```

Comment:
it doesn't hurt to have that many but what I intended moreso was more to be like 2.9995 but apparently I was tired